extensions/markdown-language-features/src/commands/showPreview.ts

```diff
@@ -68,6 +68,21 @@ export class ShowPreviewCommand implements Command {
 	}
 }
 
+export class CopyImageCommand implements Command {
+	public readonly id = 'markdown.copyImage';
```

Comment:
Command ids that are only meant to be used within an extension should typically start with an `_`